api/src/main/resources/workbench-api.yaml

```diff
@@ -6027,6 +6027,10 @@ definitions:
         description: Prefix of the research VM(s) that generated egress, including
           the user ID, e.g. all-of-us-260.
         type: string
+      workspace_namespace:
+        description: 'Worksapce namesapce(Terra Billing Project ID), after Terra project per
```

Comment:
typos

Reply:
we should rename project_name here as well, to clarify that it's the google project, not the billing project.

Reply:
I didn't do that because that is too hard? My main concern is how to make the sumologic query change, and make both compatible. So just use project_name, and just try to let everyone to forget about Terra billing project id. 
project for us is just AoU...

Reply:
Realistically I don't think we'll be able to forget about Terra Billing Projects when we say "project" in the short term.  So I would still like to see this change, but it can be in a future PR if it also requires Sumo-side changes.

One typo remains: migratioon

Reply:
Filed https://precisionmedicineinitiative.atlassian.net/browse/RW-6569 to track the new change. 
also fix the typo